README.rst

```diff
@@ -1,32 +1,69 @@
-This is a plugin for Galicaster that allows to control a remote camera from Galicaster GUI. The plugin is designed to work with Sony EVI-H100S/100V cameras, using an implementation of VISCA protocol in python.
+Galicaster Plugin Cameracontrol
+===============================
+
+This is a plugin for Galicaster that allows to control a remote camera from Galicaster GUI.
+
+Right now, the plugin works with the following cameras:
+
+- Sony:
+	- EVI-H100S/100V cameras, using an implementation of VISCA protocol in python.
+- Panasonic:
+	- AW-HE50
+	- AW-HE120
+	- AW-HE60
+	- AW-HE130
+	- AW-HE40
+	- AW-HE65
+	- AW-HE70
+	- AW-UE7
+
+Installation
+------------
 
 This plugin can be installed as a deb package or through pip installer:
 	- To generate deb package
 		In repository root folder execute ``sudo py2deb -r pathtorepo .``
 
-		-r pathtorepo  path to store deb files 
+		-r pathtorepo  path to store deb files
 
 		This will create the plugin deb package and debs for dependences. By default packages are stored in /tmp folder.
 
 	- To install with pip
-		In the repository root folder execute ``sudo pip install``
+		In the repository root folder execute ``sudo pip install`` or execute ``python setup.py sdist`` to generate a .tar package installable from pip
+
+Configuration examples
+----------------------
 
-		or execute ``python setup.py sdist`` to generate a .tar package installable from pip
+Sony EVI-H100S/100V
+-------------------
 
 Galicaster controls the camera using the serial port, so we need to add the user galicaster to the group dialout, like follows (otherwise galicaster would need root privileges):
 
 ``sudo usermod -a -G dialout galicaster``
 
-Also it's necessary to add this lines to conf.ini in galicaster folder to run the plugin properly:
```

Comment:
*these* lines, but anyway, even better:

The following lines must also be added to the `conf.ini` file so that the plugin runs properly.